mifospay/src/main/res/drawable/ic_up_arrow.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="512dp"
+    android:height="512dp"
+    android:viewportWidth="438.533"
+    android:viewportHeight="438.533">
```

Comment:
Can you set `android:viewportWith` and `android:viewportHeight` values to `512dp` please, both icons should be in the same size because it looks inconsistent when the size of up and down arrow changes.